shotover-proxy/tests/examples/mod.rs

```diff
@@ -0,0 +1,23 @@
+use crate::cassandra_int_tests::{assert_query_result, ResultValue};
+use crate::helpers::ShotoverManager;
+use serial_test::serial;
+use test_helpers::docker_compose::DockerCompose;
+
+#[tokio::test(flavor = "multi_thread")]
+#[serial]
+async fn test_cassandra_rewrite_peers_example() {
+    let _docker_compose =
+        DockerCompose::new("example-configs/cassandra-rewrite-peers/docker-compose.yml");
+
+    let shotover_manager = ShotoverManager::from_topology_file(
```

Comment:
why do we have a shotover manager?
Shouldnt shotover be launched by docker?

Reply:
To connect to the cassandra cluster, but I'll make the method available as a function.